src/entitysdk/util.py

```diff
@@ -50,5 +52,63 @@ def make_db_api_request(
             f"response : {response.text}"
         )
         raise EntitySDKError(message) from e
-
     return response
+
+
+def stream_paginated_request(
+    url: str,
+    *,
+    method: str,
+    json: dict | None = None,
+    parameters: dict | None = None,
+    files: dict | None = None,
+    project_context: ProjectContext,
+    token: str,
+    http_client: httpx.Client | None = None,
+    page_size: int = constants.page_size,
+    limit: int,
+) -> Iterator[dict]:
+    """Paginate a request to entitycore api.
+
+    Args:
+        url: The url to request.
+        method: The method to use.
+        json: The json to send.
+        parameters: The parameters to send.
+        files: The files to send.
```

Comment:
Do we need `files`? Or is it for generic support?

Reply:
Nope, removed.

Reply:
It should be removed from the docstring as well :)